docs/user/openstack/README.md

```diff
@@ -72,37 +117,27 @@ documentation in that repository for further details.
 
 * https://github.com/shiftstack-dev-tools/ocp-doit
 
-## OpenShift API Access
-
-All the OpenShift nodes are created in an OpenStack tenant network and as such, can't be accessed directly. The installer does not create any floating IP addresses.
-
-However, the installer does need access to the OpenShift's API as it is being deployed.
+## API Access
 
-There are two ways you can handle this.
+All the OpenShift nodes are created in an OpenStack tenant network and as such, can't be accessed directly in most openstack deployments. The installer does not create any floating IP addresses, but does need access to the OpenShift's API as it is being deployed. We will briefly explain how to set up access to the openshift api with and without floating IP addresses.
 
-### Bring Your Own Cluster IP
+### Using Floating IPs
 
-We recommend you create a floating IP address ahead of time, add the
-API record to your own DNS and let the installer use that address.
+This method allows you to attach two floating IP addresses to endpoints in OpenShift.
 
-First, create the floating IP:
+First, create a floating IP address for the API:
 
     $ openstack floating ip create <external network>
 
-Note the actual IP address. We will use `10.19.115.117` throughout this
-document.
-
 Next, add the `api.<cluster name>.<cluster domain>` and `*.apps.<cluster
```

Comment:
`*.apps.<clustername>.<cluster domain>` should point to another Floating IP, mapped to the Ingress VIP port.

Reply:
Yeah, following the steps, the `*.apps` bit here should be removed I think.

Reply:
Will clarify